samples/button_interrupt.py

```diff
@@ -34,6 +34,7 @@ def blink(channel):
         GPIO.output(led_pin_2, GPIO.HIGH)
         time.sleep(0.5)
         GPIO.output(led_pin_2, GPIO.LOW)
+        time.sleep(0.5)
```

Comment:
This shouldn't be needed, since the sample is written to assume that events occur infrequently enough that the duration between events is long enough to see the low level output value.

Reply:
I think it is possible to recognize that events occur, but README says "rapidly blinks the second LED five times only when the button is pressed." and it is difficult to recognize "five times" to blink.

Reply:
Oh right, this is in a loop, so this looks OK.